git_pile/git_pile.py

```diff
@@ -1952,7 +1651,7 @@ def run(self):
                             if pre_genbranch_exec:
                                 pre_genbranch_exec("", env=hook_env)
 
-                            _genbranch(root, piledir, config, genbranch_args)
+                            genbranch(root, piledir, config, genbranch_args)
```

Comment:
main source of issues with changes to genbranch in the past has been the git pile am -g. Can you add a test to make sure this code move doesn't break it?

Reply:
I was a bit confused since this part of the code is for genlinear branch and we already have tests for it. However, looking at the coverage report, I see that the path for the `-g` option to `git pile am` is indeed not covered. I'll add a test then.

Reply:
Done.